client/sites/settings/controller.tsx

```diff
@@ -60,27 +59,18 @@ export function SettingsSidebar() {
 	);
 }
 
-export async function redirectToHostingConfigIfDuplicatedViewsDisabled(
-	context: PageJSContext,
-	next: () => void
-) {
-	const { getState, dispatch } = context.store;
-	const isUntangled = await isRemoveDuplicateViewsExperimentEnabled( getState, dispatch );
+export async function redirectToHostingConfig( context: PageJSContext ) {
```

Comment:
I think we're confusing the logic here. This whole function should be just deleted; we shuold never redirect to `/hosting-config` anymore.

Reply:
Nice catch! I've mistaken `! isUntangled` with the same thing but without the negation hehe.